freeze.sh

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
+
+set -e
+set -x
+
+source manifest
+
+# Allow multiple downloads
+sed -i '/ParallelDownloads/s/^#//g' /etc/pacman.conf
+# set archive date if specified and force update/downgrade builder
+if [ -n "${ARCHIVE_DATE}" ]; then
```

Comment:
Maybe the freeze by `ARCHIVE_DATE` and the installation steps should be a different executable.
How the final archive freeze to date is handled in this scheme?